iib/web/api_v1.py

```diff
@@ -90,6 +90,16 @@ def get_builds():
     return flask.jsonify(response)
 
 
+@api_v1.route('/healthcheck')
+def get_healthcheck():
+    """
+    Respond to a health check.
+
+    :rtype: str
+    """
+    return "Health check OK"
```

Comment:
This is misleading since this API endpoint actually performs no checks. It doesn't verify that the database connection is successful or that it can communicate with the workers.

Even getting the status code of `/api/v1/builds?per_page=0` would be quick and provide a more useful indicator for health.

Reply:
I added some checks.

Reply:
Every response from API is in JSON format except this one. I would suggest to response in JSON format if it is possible.

Reply:
The semaphore service performs a regex check on the response message. Text response works much better for this.

Reply:
Do you know what mime type sends semaphore in accept header? If it's expecting text/plain we can send output based on that

Reply:
I changed it to JSON and loosened the semaphore regex a bit. It shouldn't be an issue